monitoring/configuration/sm-ks-controller-manager.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: k8s-controller-monitor
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 5s
+    path: /metrics
+    port: https
+    scheme: https
+    tlsConfig:
+      insecureSkipVerify: true
```

Comment:
Why not use the CA cert that is also available?

Reply:
The available certs are not available in the prometheus namespace. Hence, for configuration simplicity, it's better not to use them.

Reply:
I am not sure I understand. There is a CA certificate in `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`, right? What goes wrong if that one is used?

Reply:
I found the reason. A little experimentation shows that the RBAC proxy server's certificate is issued not by the cluster CA but rather by a special CA hacked up just for that pod.